db/migrate/20180329141018_add_last_update_stdout_to_configuration_script_source.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastUpdateStdoutToConfigurationScriptSource < ActiveRecord::Migration[5.0]
+  def change
+    add_column :configuration_script_sources, :last_update_stdout, :text
```

Comment:
@bdunne Aside from my naming questions in https://github.com/ManageIQ/manageiq/pull/17290#issuecomment-395879777, does this even make sense on configuration_script_sources?  I thought sources was the content of the configuration script.  But content is not invocation, and I'd expect an "output" field on the "runtime invocation" model.

Reply:
See https://github.com/ManageIQ/manageiq/pull/17290#issuecomment-395887146 maybe we should continue the conversation over there?

Reply:
We still store some data concerning the last update: status and timestamp. The only difference is that in Tower these are propagated to the project itself, so they are accessible even without explicitly asking for the last update resource.